python_tools.spec

```diff
@@ -137,7 +137,6 @@ Requires: py2-matplotlib
 Requires: py2-numpy-toolfile
 Requires: py2-sqlalchemy
 Requires: py2-pygithub
-Requires: py2-dablooms-toolfile
```

Comment:
we need a cmssw PR to go with this change. It will fail PR tests and IBs due to missing dablooms

Reply:
I am working on a coordinated change to StaticAnalyzers.